### PR TITLE
Added Zeit Now docs

### PR DESCRIPTION
**Pull request by mcollina**
Added docs for deploying on Zeit now.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Serverless.md [hunk 1]
```diff
@@ -190,3 +191,73 @@ gcloud beta run deploy --image gcr.io/PROJECT-ID/APP-NAME --platform managed
...
+
+```js
+'use strict'
+
+const build = require('./index')
+
+const app = build()
+
+module.exports = async function (req, res) {
+  await app.ready()
```
Comment (Eomm) [inline docs/Serverless.md]: should we run `ready` at every request?

File: docs/Serverless.md [hunk 2]
```diff
@@ -190,3 +191,73 @@ gcloud beta run deploy --image gcr.io/PROJECT-ID/APP-NAME --platform managed
...
+
+## Zeit Now
+
+[now](https://zeit.co/home) provides zero configuration deployment for
+Node.js applications. In order to use now, it is as simple as
+configuring your `now.json` file like the following:
+
+```json
+{
+  "builds": [
```
Comment (Eomm) [inline docs/Serverless.md]: Without version 2 the `now dev` doesn't work
```suggestion
  "version": 2,
  "builds": [
```

File: docs/Serverless.md [hunk 3]
```diff
@@ -190,3 +191,73 @@ gcloud beta run deploy --image gcr.io/PROJECT-ID/APP-NAME --platform managed
...
+
+const app = build()
+
+module.exports = async function (req, res) {
+  await app.ready()
+  app.server.emit('request', req, res)
+}
+```
+
+And a `index.js` file:
```
Comment (Eomm) [inline docs/Serverless.md]: This file should be "moved" to the directory `/api/index.js`

File: docs/Serverless.md (hunk 1 above)
Comment (mcollina) [reply docs/Serverless.md]: AWS Lambda will reuse the same process multiple times. As a result, the promise returned by `ready()` will be resolved immediately. My check is that most of our overhead come Ajv startup, see the discussion in #1598.

File: docs/Serverless.md (hunk 3 above)
Comment (mcollina) [reply docs/Serverless.md]: It doesn't. The `build` section does not include `/api`. I'm happy to add it to both if you prefer.

File: docs/Serverless.md (hunk 1 above)
Comment (Eomm) [reply docs/Serverless.md]: Thank you both for the clarification 👍

File: docs/Serverless.md (hunk 3 above)
Comment (Eomm) [reply docs/Serverless.md]: I didn't write all the info:
you have right, the example works, but if a user runs `now dev` it doesn't work without that structure so, I have afraid we could have some issue about that.

**Comment by mcollina** [comment]
@Eomm would you mind checking?

**Pull request merged**